publishing.gradle

```diff
@@ -0,0 +1,18 @@
+publishing {
+    publications {
+        gdxTinyVG(MavenPublication) {
```

Comment:
This name should probably not be gdxTinyVG.

Reply:
Hate that you need to define a name there, always forget to change it :D